### ISSUE TITLE
Fix task 6 description

### ISSUE THREAD

**Issue by mateussantana** [LaravelDaily/Test-Laravel-Validation#58]
In the README description, task number 6 is described to `"...validation rules of name/title as required fields."` when test checks for name and **description** fields. Is that a description error?

This #57  fix this description on README file!

**Comment by JakyeRU** [comment]
Yup, it's most likely a typo. I had to check the `test_form_request_validation` method because my test was failing and I didn't know why.

**Comment by ozmos** [comment]
Just discovered this issue myself
